keyboards/anne_pro/anne_pro_lighting.c

```diff
@@ -192,3 +196,35 @@ void anne_pro_lighting_set_keys(uint8_t keys, uint8_t *payload) {
     uart_tx_ringbuf_write(&led_uart_ringbuf, 5, buf);
     uart_tx_ringbuf_write(&led_uart_ringbuf, 5 * keys, payload);
 }
+
+/* Set Caps Lock red led on */
+void anne_pro_lighting_caps_lock_on(){
+		if(!leds_enabled && !not_init){
```

Comment:
I'm not sure why you would need this, as you can turn on the leds with `anne_pro_lighting_on` when `leds_enabled` is false. Otherwise it you should just be able to change the lighting.

Reply:
This part is for when you want lights off, and still be able to see caps lock.
When the leds are off, if you want to still have the caps lock indicator, you have to wake up the mcu(as in your code when you call `anne_pro_lighting_off` mcu goes to sleep), if i use `anne_pro_lighting_on` all lights get on and they should not as you want the lights off.